composer.json

```
@@ -7,7 +7,7 @@
     "ext-openssl": "*",
     "ext-soap": "*",
     "google/auth": "^1.0.0",
-    "guzzlehttp/guzzle": "^6.0",
+    "guzzlehttp/guzzle": "^7.0",
```

You could allow 6 or 7 as a version and not require people to upgrade 

```suggestion
    "guzzlehttp/guzzle": "^6.0 || ^7.0",
```

---

I'll need to check to make sure that the other changes I made to add compatibility for v7 are backwards compatible with v6, but I don't think it should be an issue. I'll check it and submit any changes a little later this afternoon 👍